crates/prune/src/metrics.rs

```diff
@@ -0,0 +1,36 @@
+use reth_metrics::{metrics, metrics::Histogram, Metrics};
+use reth_primitives::PrunePart;
+use std::collections::HashMap;
+
+#[derive(Debug, Default)]
+pub(crate) struct Metrics {
+    pub(crate) pruner: PrunerMetrics,
+    prune_parts: HashMap<PrunePart, PrunerPartMetrics>,
```

Comment:
we could flatten this but it is probably not worth it

Reply:
I mean we could add `PrunerPartMetrics` for all PrunePart variants, but not sure worth the effort